cdn/dovetail-cdn.yml

```diff
@@ -49,6 +49,27 @@ Resources:
       DistributionConfig:
         Aliases:
           - !Ref DistributionDomain
+        CustomErrorResponses:
+          # unrecognized uri, and s3 object not found
```

Comment:
If a path doesn't look like `/<program>/<digest>/<filename>`, the lambda just passes that through.  Which triggers a 403 from S3 ... and this translates it back to a 404 before returning to the user.